commands/runner.go

```diff
@@ -188,3 +198,26 @@ func splitAliasCmd(cmd string) ([]string, error) {
 
 	return words, nil
 }
+
+func getListOfBuiltInCommands() map[string]bool {
+	gitHelpOutput, _ := cmd.New("git help -a").CombinedOutput()
+	builtInCommands := parseGitHelpOutput(gitHelpOutput)
+	for hubCommand, _ := range CmdRunner.All() {
+		builtInCommands[hubCommand] = true
+	}
+	return builtInCommands
+}
+
+func parseGitHelpOutput(gitHelpOutput string) map[string]bool {
```

Comment:
I checked this. gitOutput trims the spaces in command output. Thus, we can not differentiate command list from gitOutput response. I can make a non-trimming version of gitOutput which we can use. Let me know your thoughts?